azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -191,6 +199,10 @@ public void setSubmitTime(long submitTime) {
     this.submitTime = submitTime;
   }
 
+  public void setSlaOptions(List<SlaOption> slaOptions) {
+    this.slaOptions = slaOptions;
```

Comment:
is builder pattern preferred here?
public ExecutableFlow setSlaOptions() {
   this.slaOptions = slaOptions;
   return this;
}